csm_web/frontend/src/components/Policies.tsx

```diff
@@ -0,0 +1,209 @@
+import React from "react";
+import SECTIONS from "../../static/frontend/data/enrollment_policies";
+
+export default class Policies extends React.Component {
```

Comment:
convert Policies from a class component to a functional component since we're trying to migrate to those.

Reply:
very functional now pog